admin-ui/app/components/SVG/menu/Scripts.tsx

```
@@ -0,0 +1,31 @@
+const ScriptsIcon: React.FC<ScriptsProps> = ({ fill = 'currentColor', className, style }) => {
+  return (
+    <div className={className} style={style}>
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 24 24"
+        stroke={fill}
+        strokeWidth="1.5"
+        width="28"
+        height="28"
+      >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)